Modules/Registration/PDEDeformable/include/itkMultiResolutionPDEDeformableRegistration.h

```diff
@@ -287,7 +291,7 @@ class ITK_TEMPLATE_EXPORT MultiResolutionPDEDeformableRegistration
 } // end namespace itk
 
 #ifndef ITK_MANUAL_INSTANTIATION
-#  include "itkMultiResolutionPDEDeformableRegistration.hxx"
+#include "itkMultiResolutionPDEDeformableRegistration.hxx"
```

Comment:
I think the new style calls for the old indentation. Did you run clang-format before committing your changes?